projects/mstable-v2/index.js

```diff
@@ -0,0 +1,42 @@
+const {CONFIG_DATA} = require("./config");
+const {DHEDGE_FACTORY_ABI, MSTABLE_POOL_ABI} = require("./abis");
+
+
+async function tvl(api) {
+    const { chain, } = api
+    const { dhedgeFactory, mstableManager } = CONFIG_DATA[chain];
+
+    const pools = await api.call({
+        abi: DHEDGE_FACTORY_ABI,
+        target: dhedgeFactory,
+        params: [mstableManager],
+    });
+
+    const poolSummariesRes = await api.multiCall({
+        abi: MSTABLE_POOL_ABI,
+        calls: pools,
+        permitFailure: true
+    });
+
+    const poolSummaries = poolSummariesRes.filter(i => i && i.totalFundValue !== null && i.totalFundValue !== undefined);
+
+    const totalValue = poolSummaries.reduce(
+        (acc, i) => acc + +i.totalFundValue,
+        0
+    );
+
+    return {
+        tether: totalValue / 1e18,
```

Comment:
can we count token balances in the pools instead?

Reply:
Pools can contain comples positions like aave lending v3 or pendle, not only base assets. So its better to show only total pool usd value.

Reply:
Oh, I see, but could you please add it to new page "mStable V2" astotal usd values?

Reply:
Total should be about 4M $, see: https://dhedge.org/user/0x3dd46846eed8D147841AE162C8425c08BD8E1b41